### PR TITLE
Started tests, added codecov, made quality change

### PR DESCRIPTION
**Pull request by PaDun**
Still need to figure out how to pull changes from prestoncarman's fork into mine.

### CODE REVIEW

**Comment by PaDun** [comment]
My tests seem to be working right, though a couple other tests that were already there seem to be struggling with whatever I've done.

**Comment by N-Perry** [comment]
looks good! Two Cyclomatic Complexity quality issues improved, and one cognitive complexity issue improved. Really impressive stuff Patrick! strong work

File: tests/aswwu/test_searchHandler.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+            'username':'test.profile1',
+            'views':'1'
+        }]}
+    with profile(peopledb_conn, list(gen_profiles(number = 2))):
+        url = 'http://127.0.0.1:8888/search/1718/gender=female;majors=Software'
+        resp = requests.get(url)
+    assert (resp.status_code == 200)
+    assert (json.loads(resp.text) == expected_data)
+
+#def test_views(testing_server):
```
Comment (prestoncarman) [inline tests/aswwu/test_searchHandler.py]: Remove commented out code.

File: tests/utils.py [hunk 2]
```diff
@@ -164,9 +167,10 @@ def gen_profiles(number=5):
                 "id" : 100 + i,
                 "wwuid": 9000000 + i,
                 "photo": "profiles/1617/00958-2019687.jpg",
```
Comment (prestoncarman) [inline tests/utils.py]: Why does the current year people generator use an old year?

File: tests/aswwu/test_searchHandler.py [hunk 3]
```diff
@@ -0,0 +1,222 @@
+import requests
+import json
+from tests.utils import profile, archived_profile, gen_profiles
+
+
+def test_search_specific_current(testing_server, peopledb_conn):
+    expected_data = {'results': [{
+                    'email':'None',
```
Comment (prestoncarman) [inline tests/aswwu/test_searchHandler.py]: Format these arrays like the ones below. Be consistent.

File: tests/aswwu/test_searchHandler.py [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+    with profile(peopledb_conn, list(gen_profiles(number = 3))):
+        url = "http://127.0.0.1:8888/search/1718/test.profile1"
+        resp = requests.get(url)
+    assert (resp.status_code == 200)
+    assert (json.loads(resp.text) == expected_data)
+
+
+def test_search_specific_archive(testing_server, archivesdb_conn):
+    expected_data = {'results': [{
+                    'email':'None',
```
Comment (prestoncarman) [inline tests/aswwu/test_searchHandler.py]: format array.

File: tests/aswwu/test_searchHandler.py (hunk 3 above)
Comment (PaDun) [reply tests/aswwu/test_searchHandler.py]: Done

File: tests/aswwu/test_searchHandler.py (hunk 4 above)
Comment (PaDun) [reply tests/aswwu/test_searchHandler.py]: Done

File: tests/aswwu/test_searchHandler.py (hunk 1 above)
Comment (PaDun) [reply tests/aswwu/test_searchHandler.py]: Done

File: tests/utils.py (hunk 2 above)
Comment (PaDun) [reply tests/utils.py]: The computers are lazy and don't want to update their photos, perhaps?
I'll take it off.

**Pull request merged**